scripts/gha/build_testapps.py

```diff
@@ -426,6 +426,12 @@ def setup_unity_project(dir_helper, setup_options):
   _copy_unity_assets(dir_helper, setup_options.testapp_file_filters)
   _add_menu_scene(dir_helper)
   _add_automated_test_runner(dir_helper)
+  # If running on CI, copy over the file that defines that custom scripting symbol
+  # For more info, see: https://docs.unity3d.com/Manual/CustomScriptingSymbols.html
```

Comment:
It is also possible to use [PlayerSettings.SetScriptingDefineSymbolsForGroup](https://docs.unity3d.com/ScriptReference/PlayerSettings.SetScriptingDefineSymbolsForGroup.html) .
This is what we did in the past: https://github.com/firebase/firebase-unity-sdk/blob/main/scripts/gha/integration_testing/AppBuilderHelper.cs
But you can decide how to implement it.

Reply:
I guess for now we can submit this, and in the future if we have the need for more, or this does have some issue, we can look into the other method.